src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,352 @@
+        if (this.isChecked) {
+          this.$_requestWakeLock();
+        } else {
+          this.$_releaseWakeLock();
+        }
+      },
+      $_requestWakeLock: async function() {
+        try {
+          this.wakeLock = await navigator.wakeLock.request('screen');
+          //Start new timer
+          this.$_handleAbortTimer();
+          //Only add event listeners after wake lock is successfully enabled
+          document.addEventListener(
+            'visibilitychange',
+            this.$_handleVisibilityChange,
+          );
+          window.addEventListener(
+            'scroll',
+            debounce(this.$_handleAbortTimer, scrollDebounceMs),
+          );
+          this.$_raiseAnalyticsEvent('Wake Lock Toggle Enabled');
+          if (window.GeneralMills.debugMode) {
+            console.log('Wake Lock Active');
+          }
+        } catch (e) {
+          this.isChecked = false;
+          if (window.GeneralMills.debugMode) {
+            console.log(`Wake Lock Request Error: ${e.name}, ${e.message}`);
+          }
+        }
+      },
+      $_releaseWakeLock: function() {
+        try {
+          this.wakeLock.release();
+          this.wakeLock = null;
+          //Clear timer
+          this.$_handleAbortTimer();
+          //Clean up event listeners
+          document.removeEventListener(
+            'visibilitychange',
+            this.$_handleVisibilityChange,
+          );
+          window.removeEventListener(
+            'scroll',
+            debounce(this.$_handleAbortTimer, scrollDebounceMs),
+          );
+          if (window.GeneralMills.debugMode) {
+            console.log('Wake Lock Released');
+          }
+        } catch (e) {
+          if (window.GeneralMills.debugMode) {
+            console.log(`Wake Lock Release Error: ${e.name}, ${e.message}`);
+          }
+        }
+      },
+      $_handleAbortTimer: function() {
+        //If there is an existing timer then clear it and set to zero
+        if (this.timerId !== 0) {
+          clearTimeout(this.timerId);
+          this.timerId = 0;
+        }
+        //Start new timer; Will be triggered from toggle enabled or scroll event
+        if (this.isChecked) {
+          this.timerId = setTimeout(
+            this.$_releaseWakeLock,
+            this.settingsInternal.timeoutDurationMs,
+          );
+        }
+      },
+      $_handleVisibilityChange: function() {
+        //Handle navigating away from page/tab
+        if (this.isChecked) {
+          this.$_releaseWakeLock();
+          this.isChecked = false;
+        }
+      },
+      $_raiseAnalyticsEvent: function(eventType) {
+        let eventParams = {
+          EventType: eventType,
+          Position: window.location.pathname || '',
+        };
+        GeneralMills.Analytics && GeneralMills.Analytics.raiseEvent(eventParams);
+      },
+    },
+  };
+</script>
+```
+
+### Results
+
+The Vue.js component has been deployed on the three sites and delivered great results.
```

Comment:
The Vue.js component has been deployed on all three sites and delivered great results. During the period from December 10th, 2019 to January 10th, 2020, BettyCrocker.com reported the following metrics: